src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1347,6 +1358,32 @@ void Transformations::MainSnippets() {
                 return is_unsupported_parallel_work_amount(n, pshape);
             },
             snippets::pass::TokenizeMHASnippets);
+        CPU_SET_CALLBACK_X64(
+            snippetsManager,
+            [&](const std::shared_ptr<const ov::Node>& n) -> bool {
+                if (!is_supported_matmul(n))
+                    return true;
+                if (is_unsupported_parallel_work_amount(n, n->get_output_partial_shape(0)))
+                    return true;
+
+                // TODO: release these conditions
```

Comment:
Could you please add the comment that we tested only on small models? And due to this fact we tokenize sequence with small shapes only. To not forget what is the reason of these conditions :)

Reply:
Done